web-test-runner.config.mjs

```diff
@@ -85,11 +78,12 @@ export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
     esbuildPlugin({ ts: true, target: 'esnext'}),
   ],
 
-  testRunnerHtml: testFramework =>
-    `<html>
+  testRunnerHtml: testFramework => {
+    return `<html>
       <head><link rel="stylesheet" href="./dist/themes/default.css"></head>
       <body>
         <script type="module" src="${testFramework}"></script>
       </body>
-    </html>`,
+    </html>`;
```

Comment:
We can revert this part. Apparently, I forgot to revert it while trying out mocking icon component. 😊